CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ add_library(direwolf
     src/renderengine.cpp
     include/direwolf/renderengine.h
     # TODO: this shouldn't be here.. remove when vulkan stuff is included in the direwolf wrappers
+    include/direwolf/vulkan/vulkancommons.h
```

Comment:
NIT: vulkancommon or vulkanutils imo

Reply:
Well.. they have different purposes, but agreed. Vulkan Utils should probably be removed.
The way it is now, commons should be inclued in a header, utils in a source file